src/app/state/wallet/saga.test.ts

```diff
@@ -70,6 +71,40 @@ describe('Wallet Sagas', () => {
         .silentRun(50)
     })
 
+    it('Should open from ledger and select the first imported wallet as active', () => {
```

Comment:
Can this test be extended to validate the bug was fixed?

Reply:
Looks like you have already answered this question :D